.github/workflows/docker-hardfork-ci.yml

```diff
@@ -3,19 +3,12 @@ name: Docker Hardfork CI
 on:
   push:
     branches:
-    - 'develop'
-    - 'release_hf_*'
-    paths:
-      - 'libraries/chain/hardfork.d/*'
-      - '.github/workflows/docker-hardfork-ci.yml'
-
+      - '*'
 
 jobs:
   build-testnet:
     runs-on: self-hosted
 
-    if: github.ref == 'refs/heads/develop'
```

Comment:
why not?

Reply:
I would like to run CI on push to feature branch.